### PR TITLE
Blog post 2017-05-09-Time-scheduling-with-Chime.md

### PR DESCRIPTION
**Pull request by LisiLisenok**

### CODE REVIEW

File: src/site/blog/posts/2017-05-09-Time-scheduling-with-Chime.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+title: Time scheduling with Chime
+template: post.html
+date: 2017-05-09
+author: LisiLisenok
+draft: true
+---
+
+## Time scheduling.  
+
+Vert.x executes periodic and delayed actions with
```
Comment (vietj) [inline src/site/blog/posts/2017-05-09-Time-scheduling-with-Chime.md]: should be Eclipse Vert.x

**Comment by LisiLisenok** [comment]
@vietj are you agree to publish this?

**Comment by vietj** [comment]
yes

**Comment by vietj** [comment]
will do now

**Pull request merged**